packages/utils/src/createCooldown.ts

```diff
@@ -0,0 +1,15 @@
+/**
+ * Function returned by `createCooldown` returns `true` only when previous `true` was returned
+ * `cooldownMs` or more millis ago, and `false` otherwise. First call always returns `true`.
+ */
+export const createCooldown = (cooldownMs: number) => {
```

Comment:
nit: for more clarity when using this could work better?

```
export const createCooldown = (cooldownMs: number) => {
    let last = 0;
    return {
        getIsCooledDown() {
            const now = Date.now();
            if (now - last >= cooldownMs) {
                last = now;
                return true;
            }
            return false;
        },
    };
};
```

Reply:
Dunno, I kinda like the simplicity of
```
const shouldDoSomething = createCooldown(1000);
...
if (shouldDoSomething()) doIt();
```